.github/workflows/pull-request-checks.yaml

```diff
@@ -287,7 +287,7 @@ jobs:
           submodules: recursive
       - name: Fetch dependencies
         run: |
-          choco install winflexbison3 strawberryperl
+          choco install winflexbison3 strawberryperl z3
```

Comment:
Should this commit be dropped?

Reply:
This commit exists, so that I am not doing both adding and updating in subsequent commits.